packages/components/dialog/src/dialog.ts

```diff
@@ -14,50 +18,89 @@ import styles from './dialog.scss.js';
  * @slot default - Body content for the dialog
  * @slot footer - Footer content for the dialog
  * @slot header - Header content for the dialog
+ * @slot close - Closing button (placed in header) for the dialog
+ * @slot header-buttons - More space for buttons for the dialog's header
  * @slot title - The title of the dialog
+ * @slot subtitle - The subtitle of the dialog
  */
 export class Dialog extends ScopedElementsMixin(LitElement) {
   /** @private */
   static get scopedElements(): ScopedElementsMap {
     return {
-      'sl-button-bar': ButtonBar
+      'sl-button': Button,
+      'sl-button-bar': ButtonBar,
+      'sl-icon': Icon
     };
   }
 
   /** @private */
-  static override styles: CSSResultGroup = styles;
+  static override styles: CSSResultGroup = [breakpoints, styles];
 
+  /** @private */
   @query('dialog') dialog?: HTMLDialogElement;
 
   /** Disables the ability to close the dialog using the Escape key. */
   @property({ type: Boolean, attribute: 'disable-close' }) disableClose = false;
 
+  /** Determines whether closing button (default one) should be shown in the top right corner. */
+  @property({ type: Boolean, attribute: 'closing-button' }) closingButton?: boolean;
+
   /** The ARIA role of the dialog. */
   @property() override role: 'dialog' | 'alertdialog' = 'dialog';
 
+  /** Alignment of the action buttons in the dialog footer
+   * @type {'start' | 'end'} */
+  @property({ reflect: true, attribute: 'buttons-align' }) buttonsAlign: Extract<ButtonBarAlign, 'start' | 'end'> =
```

Comment:
Not sure if `buttons-align` is correct, or if it should be `align-buttons`. The latter feels more natural.

Reply:
What's the idea behind this property btw? Why would a developer change this setting? Or is this a theme variable? If the latter is the case, then having (`static`) dialog defaults would be better I think.

You could also remove the property and simply have the developer slot his/her own `<sl-button-bar>` in the `footer` slot?

Reply:
We have the acceptance criteria
<img width="482" alt="image" src="https://github.com/sl-design-system/components/assets/111562742/c0275548-e7d1-499c-a16e-1ed3b3502ada">

that in the footer we can have buttons on the left and on the right side. It doesn't come from the token. So do we want to keep it as it is or do you suggest that we should let the developer using `sl-button-bar` on his/her own in the `footer` slot?

Reply:
The possibilities of the footer usage we can find in Figma: https://www.figma.com/file/1oIVcD9VMbOtLXYp48ehsA/Sanoma-Learning-Light-mode?type=design&node-id=4801-22057&mode=dev

We have now two slots inside `sl-button-bar` .